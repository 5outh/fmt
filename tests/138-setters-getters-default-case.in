<?php
//passes:TwoCommandsInSameLine,SettersAndGettersPass|invalid,ConstructorPass|invalid,OrderUseClauses,AddMissingCurlyBraces,SmartLnAfterCurlyOpen,NormalizeLnAndLtrimLines,MergeParenCloseWithCurlyOpen,MergeCurlyCloseAndDoWhile,MergeDoubleArrowAndArray,ExtraCommaInArray,ResizeSpaces,Reindent,ReindentColonBlocks,ReindentLoopColonBlocks,ReindentIfColonBlocks,ReindentObjOps,EliminateDuplicatedEmptyLines,AlignEquals,AlignDoubleArrow,LeftAlignComment,RTrim,OrderMethod,NormalizeLnAndLtrimLines,Reindent


class A {
	private $a;
	public $b;
	protected $c;

	public function setA($a){
		$this->a = $a;
	}
}
