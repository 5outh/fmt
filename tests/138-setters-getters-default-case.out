<?php
//passes:TwoCommandsInSameLine,SettersAndGettersPass|invalid,ConstructorPass|invalid,OrderUseClauses,AddMissingCurlyBraces,SmartLnAfterCurlyOpen,NormalizeLnAndLtrimLines,MergeParenCloseWithCurlyOpen,MergeCurlyCloseAndDoWhile,MergeDoubleArrowAndArray,ExtraCommaInArray,ResizeSpaces,Reindent,ReindentColonBlocks,ReindentLoopColonBlocks,ReindentIfColonBlocks,ReindentObjOps,EliminateDuplicatedEmptyLines,AlignEquals,AlignDoubleArrow,LeftAlignComment,RTrim,OrderMethod,NormalizeLnAndLtrimLines,Reindent

class A {
	private $a;
	public $b;
	protected $c;
	
	public function getB() {
		return $this->b;
	}
	protected function getC() {
		return $this->c;
	}
	public function setA($a) {
		$this->a = $a;
	}
	public function setB($b) {
		$this->b = $b;
	}
	protected function setC($c) {
		$this->c = $c;
	}
	function __construct($a, $b, $c) {
		$this->setA($a);
		$this->setB($b);
		$this->setC($c);
	}
}
